tests/functional/storageclass/test_ec_default_pool_and_sc.py

```
@@ -0,0 +1,262 @@
+
+        bound_sc = pvc_obj.get().get("spec").get("storageClassName")
+        log.info(f"PVC bound to StorageClass: {bound_sc}")
+        assert bound_sc == rbd_sc_name, (
+            f"PVC without storageClassName bound to '{bound_sc}', "
+            f"expected default EC RBD SC '{rbd_sc_name}'"
+        )
+
+        # Step 5: Capture baseline EC data pool usage
+        metadata_pool = helpers.default_ceph_block_pool()
+        data_pool = helpers.get_data_pool_name()
+        log.info(f"Metadata pool: {metadata_pool}, Data pool (EC): {data_pool}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)